src/Authentication/Authentication.Core/Utilities/AuthenticationHelpers.cs

```diff
@@ -59,16 +63,23 @@ private static async Task<TokenCredential> GetManagedIdentityCredentialAsync(IAu
 
         private static async Task<InteractiveBrowserCredential> GetInteractiveBrowserCredentialAsync(IAuthContext authContext, CancellationToken cancellationToken = default)
         {
+            using AzureEventSourceListener listener = AzureEventSourceListener.CreateConsoleLogger();
```

Comment:
Let's remove the Azure events listener now that we've resolved the missing dll issue.
```suggestion
```

Reply:
Noted, this would be removed when WAM has been tested before merging with features/2.0